espflash/src/error.rs

```diff
@@ -554,6 +554,7 @@ impl RomError {
     }
 }
 
+#[cfg(feature = "cli")]
```

Comment:
Not particularly fond of this, but we get a dead code warning if we don't. Better ideas are most welcome here :D